### DISCUSSION TITLE
Dashboard - mkdir(): Permission denied

### DISCUSSION THREAD

**Discussion by swegal** [automadcms/discussions#42] [Q&A]
Hey,

I just set up a new installation of the latest version on my web server. The landing page looks fine with "Welcome
You have successfully installed Automad 2. " But as soon as I press on "Dashboard" button I get into a repetitive loading error with a small red popup message in the right hand upper corner with "mkdir(): Permission denied".

Can anyone tell me maybe what's causing this? Any help very much appreciated!

**Comment by marcantondahmen** [comment]
How did you install it? The installation directory and its subdirectories must be writable by PHP.

**Comment by cracksilver** [comment]
Did you copy all hidden files?

**Comment by fighter2023** [comment]
In ProjectInstaller.php line 69:

  mkdir(): Permission denied
